helm/cluster/ci/test-cgroupsv1-values.yaml

```diff
@@ -18,6 +18,10 @@ global:
     def02:
       cgroupsv1: false
       replicas: 3
+internal:
+  ephemeralConfiguration:
+    offlineTesting:
+      renderWithoutReleaseResource: true
```

Comment:
What changed here? Can you explain why this is needed now? Because I think this might cause other places to no longer render as expected.

Reply:
I can't figure out why the conftests started failing now without it - https://app.circleci.com/pipelines/github/giantswarm/cluster/2076/workflows/5c5efbda-f6ec-4bed-8750-44811022933e/jobs/2012